client/transactions/list/index.js

```diff
@@ -247,31 +247,27 @@ export const TransactionsList = ( props ) => {
 		);
 	} );
 
-	const summaryCurrency =
-		0 < transactions.length
-			? transactions[ 0 ].currency.toUpperCase()
-			: 'USD';
```

Comment:
As with my comment on the server PR, I'm wondering if this needs to change? At least in the context of the deposit screen, we know that the list of transactions will all be of the same currency.

Reply:
That's was my original approach, but revisiting it after some time doesn't look any good. I'm convinced that the summary should come from the server already with currency provided.